user-service/src/main/java/org/java/mentorship/user/domain/UserEntity.java

```diff
@@ -0,0 +1,28 @@
+package org.java.mentorship.user.domain;
+
+import lombok.Data;
+import org.java.mentorship.contracts.user.dto.User;
+
+@Data
+public class UserEntity {
+    private Integer id;
+
+    private String email;
+    private String firstName;
+    private String lastName;
+    private Boolean verified;
+    private String hashedPassword;
+    private String verificationToken;
+
+    public User ToContract() {
```

Comment:
The DTO class shouldn't contain mapping loggic
If you want to map it to a different class, you should create a Mapper class for it (class that implements the Mapper design pattern)

Reply:
Should I put it in a user.domain.mapper package?